active_utils.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
 
   s.rubyforge_project = "active_utils"
 
-  s.add_dependency('activesupport', '>= 3.2')
+  s.add_dependency('activesupport', '>= 3.2', '< 5.1.0')
```

Comment:
We should, but we cannot time the 5.1 release (actually maybe we can these days :). I've had it happen to me in the post where this came up at the worst possible times (I was on vacation). For a gem managed by a full team that's probably less of an issue.